website/docs/r/web_application_firewall_policy.html.markdown

```diff
@@ -185,9 +185,9 @@ The `exclusion` block supports the following:
 
 The `managed_rule_set` block supports the following:
 
-* `type` - (Optional) The rule set type.
+* `type` - (Optional) The rule set type. Possible values: `OWASP`, `Microsoft_BotManagerRuleSet`.
```

Comment:
for consistency with other resources could we sort this alphabetically?

```suggestion
* `type` - (Optional) The rule set type. Possible values: `Microsoft_BotManagerRuleSet` and `OWASP`.
```

Reply:
Fixed